### ISSUE TITLE
Creating a small map window in the explore view

### ISSUE THREAD

**Issue by siddharth0248** [NASA-IMPACT/veda-ui#608]
For GHG Centre it would be really useful to have a small map window in the explore map section. e.g.
![image](https://github.com/NASA-IMPACT/veda-ui/assets/118487103/a3e6c1e1-332c-4961-a535-17c968fbe6e9)

**Comment by aboydnw** [comment]
@siddharth0248 certainly. Can you provide a little more context on where this came from? What value does this provide to a user and which workflows does this support?

**Comment by siddharth0248** [comment]
For e.g. when we try to visualize the methane plume dataset, the user doesn't get an idea in which geographic location he/she is.

**Comment by aboydnw** [comment]
@siddharth0248 I added this to the spreadsheet. I initially assigned a priority level of 3 because of the other request to toggle on/off layers seems to address a similar issue of the user having context of place, and also the workaround of simply zooming in and out on the window could hopefully be usable for scientists in the meantime. Screen real estate is really valuable in this view and I would hesitate to make this sort of change before rolling out the updated exploration view and doing some more thorough user research.

cc @Jeanne-le-Roux @deborahUAH since I added it to the spreadsheet
